src/__utils__/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+export const getArticles = () => {
+    const path = 'https://nc-news-w53u.onrender.com/api/articles'
+    return axios.get(path)
+        .then((response) => {
+            return response.data.articles
+        })
+}
+
+export const getArticle = (id) => {
+    const path = `https://nc-news-w53u.onrender.com/api/articles/${id}`
```

Comment:
Thanks for refactoring into this file! ⭐ You can actually go a step further though and have baseURL if you wanna refactor at some point 🙂